l10n_ch_import_camt054/models/fds_postfinance_file_camt.py

```diff
@@ -0,0 +1,42 @@
+from odoo import models, api
+
+import base64
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class FdsPostfinanceFileCamt(models.Model):
+    _inherit = 'fds.postfinance.file'
+
+    @api.multi
+    def import2bankStatements(self):
+
+        res = True
+        account_camt_parser_obj = self.env[
+            'account.bank.statement.import.camt.parser']
+
+        for pf_file in self:
+            #for pf_file in fds_file:
+            values = {'data_file': pf_file.data}
+            bs_import_obj = self.env['account.bank.statement.import']
+            bank_wiz_imp = bs_import_obj.create(values)
+            decoded_file = base64.b64decode(bank_wiz_imp.data_file)
```

Comment:
Lines 21-24 are useless in my opinion. decoded_file is only `base64.b64decode(pf_file.data)`